aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -265,6 +265,23 @@ To deploy the app to production, the following resources need to be provisioned:
   * There are no specific requirements for this certificate; it can be a self-signed certificate or a certificate provisioned through a CA authority.
   * It can be generated through standard tools like PowerShell or OpenSSL.
   * It can be installed into the certificate store on the target machines or deployed as a *.pfx* file with a strong password.
+### Example: Deploy to a Web Hosting
+In your web Hosting panel create or load your certificate. Then in the app's *appsettings.json* file, modify the `IdentityServer` section to include the key details:
+
+```json
+"IdentityServer": {
+  "Key": {
+    "Type": "Store",
+    "StoreName": "WebHosting",
```

Comment:
I don't think this needs to be on a "custom" store, this should be the personal user store certificate, I don't know the implications of creating a custom store, so let's rather stick to the known and tried paths.

Reply:
Hi. This custom store(webHosting) is the default store where hosting companies store website's SSL Certificates. The identity Server requires an SSL Certificate to be loaded. 
I personally used Plesk to host my ASP.NET core application with this template and after a week of trial and error, I found this combination to be working. There is no need to create a new custom store. The "webHosting" option is available to use, yet it is not mentioned in the documentation for StoreName Enum.
https://stackoverflow.com/a/31066795/11465748
Just a side note, I also tried to generate  the pfx file and publish it with the app and then read it that way, but web hosting companies usually won't allow that. Also, I tried access the uploaded certificates through "My" store name, and it didn't work either.

Reply:
Fair enough, I stand corrected :)